### DISCUSSION TITLE
Blank page after deployment

### DISCUSSION THREAD

**Discussion by jeanjcode** [vercel/community#3961] [Help]
### Summary

I created an app in React and deployed it to Vercel with no errors, but I only see a blank page. I was able to deploy to github pages but not in Vercel.

Any ideas?

### Example

https://jc-portfolio-sable.vercel.app/

### Steps to Reproduce

No error msgs

**Comment by amyegan** [comment]
Hey @jeanjcode. In the my browser's dev tools console, I see `Uncaught SyntaxError: Unexpected token '<'`.

<img width="313" alt="Console showing uncaught syntax error" src="https://github.com/vercel/community/assets/8247105/62a95c9d-2c39-4f72-a7fa-916efc292bfc">


If you can share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give us a better view of what is going wrong.

**Reply by jeanjcode** [reply]
Hi Amy: Here's a screenshot:
![Screenshot (311)](https://github.com/vercel/community/assets/143808041/9136e5fa-e0df-4644-955f-c735aec2b0e7)

**Reply by stepantsuk** [reply]
i have the same issue...

**Reply by amyegan** [reply]
Without knowing more about your project, I don't have any specific answers. But here are a few resources I found that may help you identify the cause:
- https://stackoverflow.com/questions/75368559/unexpected-token-at-of-a
- https://stackoverflow.com/questions/70702728/uncaught-syntaxerror-unexpected-token-in-react-app
- https://stackoverflow.com/questions/53957787/reactjs-unexpected-token

If you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), someone here might be able to recognize what is going wrong.

**Reply by stepantsuk** [reply]
I opened discussion https://github.com/orgs/vercel/discussions/5079
please help ))
